src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -249,6 +249,12 @@ pscanalpha.usernameidor.otherinfo = The hash was an {0}, with value: {1}
 pscanalpha.usernameidor.soln = Use per user or session indirect object references (create a temporary mapping at time of use). Or, ensure that each use of a direct object reference is tied to an authorization check to ensure the user is authorized for the requested object. 
 pscanalpha.usernameidor.refs = https://www.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References\nhttps://www.owasp.org/index.php/Testing_for_Insecure_Direct_Object_References_(OTG-AUTHZ-004)
 
+pscanrules.xaspnetversioncanner.name = X-AspNet-Version Response Header Scanner
```

Comment:
`pscanrules` needs to be changed to `pscanalpha`, otherwise ZAP will not find the messages (the prefix is "exclusive" for each add-on), missed this one :/